modules/nf-core/gatk4/markduplicates/environment.yml

```diff
@@ -4,6 +4,6 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - bioconda::gatk4=4.4.0.0
+  - bioconda::gatk4=4.5.0.0
```

Comment:
meed to update the mulled container

Reply:
https://github.com/BioContainers/multi-package-containers/pull/2934